docs/src/routes/docs/components/snackbar-container.svx

```diff
@@ -0,0 +1,114 @@
+---
+name: SnackbarContainer
+relatedComponents:
+  - name: Snackbar
+    link: ./docs/components/snackbar
+---
+
+<script>
+  import { onMount } from 'svelte';
+  import { Button, SnackbarContainer } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+  import fixAnchorLinks from 'src/utils/fix-anchor-links.js';
+
+  onMount(fixAnchorLinks);
+</script>
+
+# SnackbarContainer
+
+The wrapper element creating a context to show snackbars.
+
+The main piece of functionality is the `showSnackbar` function, which you can access through a variety of methods described below. This function accepts an object with the following fields:
+
+- `component` (default: [==`Snackbar`==](./docs/components/snackbar)) – the component to render as a snackbar;
+- `props` (default: `{}`) – the props to be passed to the snackbar. The component also injects the `closeCallback` prop into this object, which is a function that can be called to force-close the snackbar;
+- `duration` (default: `4000`) – the duration to show the snackbar in milliseconds.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <SnackbarContainer let:showSnackbar>
+      <Button on:click={() => showSnackbar({ props: { text: 'Hello!' } })}>
+        press to see
+      </Button>
+    </SnackbarContainer>
+  </div>
+  <div slot="source">
+
+```svelte
+<SnackbarContainer let:showSnackbar>
+  <Button on:click={() => showSnackbar({ props: { text: 'Hello!' } })}>
+    press to see
+  </Button>
+</SnackbarContainer>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`position`** | `SnackbarPositions.BOTTOM_LEFT` | [==`SnackbarPositions`==](#snackbar-positions) | The position of the snackbar stack inside the window. |
+
+### Class Properties
+
+All `<SnackbarContainer>` components create the `.snackbar-stack` element after the default slot content.
```

Comment:
`.snackbar-stack` is not an element name

Reply:
It's a CSS selector, I just figured that'd be enough. Any suggestions?

Reply:
It is understandable, just not really technically accurate. "... create an element with the `.snackbar-stack` class after ..." would be more accurate. (Frankly, I don't like that we write both the dot and the word "class", but we do that everywhere and it is understandable enough)